db/service_updates.yml

```diff
@@ -1,3 +1,43 @@
+- date: '2023-11-29'
+  title: Get ready to submit your school placements data in Register
+  content: |
+    You’ll be asked to include school placement data in the next performance profiles sign off in Register.
+
+    The next performance profile sign off period will start in December 2023 with a deadline of 31 January 2024.
+
+    You’ll be asked to sign off performance profile data for registered trainees with a course outcome in the 2022 to 2023 academic year.
+
+    <h3>Type of placement data you must have in Register</h3>
+
+    Each trainee with a course outcome in the 2022 to 2023 academic year must have a minimum of 2 school placements.
+
+    You’ll need to check you have the placement school URN, if known, otherwise the school name and postcode.
+
+    <h3>Next steps for SCITT training providers</h3>
+
+    If you keep a separate list or spread sheet of school placement information, make sure you include the school’s Unique Reference Number (URN).
+
+    If you do not have the URN, you’ll need the school or setting name and postcode.
+
+    <h3>Next steps for HEI training providers</h3>
+
+    We’ll import the URN for all HEI trainee placements from the student collection into Register so that you do not have to re-key this information again.
+
+    We aim to complete this by late December 2023. You do not have to do anything at this stage.
+
+    You’ll be contacted by early January 2024 to let you know when the import is complete. You can then:
+
+      <ul>
+          <li>check for any gaps in your placement information in Register</li>
+          <li>update any missing placement information in Register</li>
+      </ul>
```

Comment:
Apart from here we have referred it as 

`Placement data` vs `Placement information`

Or we collecting data or information